seedsource_core/django/seedsource/management/commands/import_vector_data.py

```diff
@@ -0,0 +1,63 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from tempfile import mkdtemp
+import shutil
+import subprocess
+import os
+
+class Command(BaseCommand):
+    help = 'Import vector data to be served as mapbox vectortiles. Viewable on front-end in "Layers" tab.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('shapefile', nargs=1, type=str)
+
+    def _write_out(self, output):
+        self.stdout.write('\033[0;33m' + output + '\033[0m')
+
+    def handle(self, shapefile, *args, **options):
+        tiles_dir = os.path.join(settings.BASE_DIR, "tiles")
+        layers_dir = os.path.join(tiles_dir, "layers")
+        file_path = os.path.abspath(shapefile[0])
+        name = os.path.splitext(os.path.basename(file_path))[0]
+
+        if not os.path.exists(layers_dir):
+            os.makedirs(layers_dir)
+
+        tmp_dir = mkdtemp()
+
+        try:
+            self._write_out(f'Converting {name} to EPSG:4326 GeoJSON for processing...')
+            subprocess.run([
+                'ogr2ogr',
+                '-f',
+                'GeoJSON',
+                '-t_srs',
+                'EPSG:4326',
+                os.path.join(tmp_dir, 'output.json'),
+                '/vsizip/' + file_path
```

Comment:
This `/vsizip/` is breaking the `ogr2ogr` command (not surprising because it results in a non-existent path!)

```
FAILURE:
Unable to open datasource `/vsizip//Users/nikmolnar/Downloads/us_eco_l3/us_eco_l3.shp' with the following drivers.
```

Reply:
Ah, I see. This is to work with zipped data. Nice! We'll also need to handle unzipped data, though.